lib/models/rules.js

```diff
@@ -33,7 +33,8 @@ var util = require('util'),
     actions = require('./actions'),
     namePattern = /^[a-zA-Z0-9_-]+$/,
     MaxNameLength = 50,
-    errors = {};
+    errors = {},
+    logger = require('logops');
```

Comment:
Better to have all the require() together. Pleace move up (just afeter `actions` declaration).

Reply:
Fixed in 9c80eb9